fdbserver/MasterProxyServer.actor.cpp

```diff
@@ -1409,7 +1459,8 @@ ACTOR Future<GetReadVersionReply> getLiveCommittedVersion(ProxyCommitData* commi
 			rep = v;
 		}
 	}
-	rep.processBusyTime = 1e6 * (g_network->isSimulated() ? deterministicRandom()->random01() : g_network->networkInfo.metrics.lastRunLoopBusyness);
+	rep.processBusyTime = FLOW_KNOBS->BASIC_LOAD_BALANCE_COMPUTE_PRECISION*std::min((std::numeric_limits<int>::max()/FLOW_KNOBS->BASIC_LOAD_BALANCE_COMPUTE_PRECISION)-1,commitData->stats.getRecentRequests());
```

Comment:
with the current knob values, if a proxy released more than 200,000 GRVs per second it would exceed this limit. This is unlikely to happen, but is possible, which is why I have the protection in place. An ASSERT would crash the proxy, so it would be bad.